src/sentry/api/event_search.py

```diff
@@ -1234,6 +1247,7 @@ def get_filter(query=None, params=None):
 FIELD_ALIASES = {
     "project": {"fields": ["project.id"], "column_alias": "project.id"},
     "issue": {"fields": ["issue.id"], "column_alias": "issue.id"},
+    "error.unhandled": {"fields": [["notHandled", [], "error.unhandled"]]},
```

Comment:
Can these keys make use of the constant aliases defined above like `ERROR_UNHANDLED_ALIAS`?